packages/app/utils/storage.ts

```diff
@@ -0,0 +1,67 @@
+import EventEmitter from 'events';
+import * as SecureStore from 'expo-secure-store';
+
+const isNextApp = process.env.NEXT_PUBLIC_APP === 'next';
```

Comment:
I think a cleaner way to handle would be to have a storage.native.ts file / storage.web.ts file. The logic can be separated across the two, and react native web automatically uses the proper file.

Reply:
Exactly, I will change it